_includes/help/nav_sidenav.html

```diff
@@ -1,20 +1,42 @@
-<nav class="sm-col-right sm-col-4 sm-show">
-  <ul class="list-reset pt2">
-    {% for question in related_questions %}
-      {% if question.url == page.url %}
-        {% assign _index = forloop.index | modulo:7 %}
-        {% if _index == 0 %}
-          {% assign _index = 1 %}
-        {% endif %}
+<div class="grid-col grid-col-3 grid-offset-2">
+  {% include search.html size="small" %}
+  <nav class="margin-y-4 sidenav">
+    <ul class="usa-accordion usa-sidenav">
+      {% for subpage in site.help_pages %}
+      {% assign subpage_slug = subpage | slugify %}
+      {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
+      {% capture is_initially_expanded %}
+      {% if page.url contains subpage_slug %}true{% else %}false
       {% endif %}
```

Comment:
Can we indent all the blocks of control statements? `for`, `if`, `capture` basically anything that requires an `end` should be indented IMO, makes it easy to follow the structure

This is just an example, looks like the `endfor` and `endcapture` are outside this section

```suggestion
      {% for subpage in site.help_pages %}
        {% assign subpage_slug = subpage | slugify %}
        {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
        {% capture is_initially_expanded %}
        {% if page.url contains subpage_slug %}true{% else %}false{% endif %}
```